mapproxy/test/system/test_wmts.py

```diff
@@ -61,6 +61,10 @@ def setup(self):
         self.common_tile_req = WMTS100TileRequest(url='/service?', param=dict(service='WMTS',
              version='1.0.0', tilerow='0', tilecol='0', tilematrix='01', tilematrixset='GLOBAL_MERCATOR',
              layer='wms_cache', format='image/jpeg', style='', request='GetTile'))
+        self.common_featureinfo_req = WMTS100FeatureInfoRequest(url='/service?', param=dict(service='WMTS',
+             version='1.0.0', tilerow='0', tilecol='0', tilematrix='01', tilematrixset='GLOBAL_MERCATOR',
+             layer='wms_cache', format='image/jpeg', style='', request='GetFeatureInfo',
+             infoformat='application/vnd.ogc.gml', i='1', j='1'))
```

Comment:
Please choose i/j and tilerow/tilecol with different values for all new tests. This makes it easier to detect errors where values/axis are swapped.
E.g. tilerow='2', tilecol='3', tilematrix='04', i='19', j='232'